src/CoreWCF.Primitives/src/CoreWCF/Description/XmlSerializerOperationBehavior.cs

```diff
@@ -34,7 +34,7 @@ public XmlSerializerOperationBehavior(OperationDescription operation, XmlSeriali
             OperationReflector = parentReflector.ReflectOperation(operation, attribute ?? new XmlSerializerFormatAttribute());
         }
 
-        internal XmlSerializerOperationBehavior(OperationDescription operation, XmlSerializerFormatAttribute attribute, Reflector parentReflector)
+        public XmlSerializerOperationBehavior(OperationDescription operation, XmlSerializerFormatAttribute attribute, Reflector parentReflector)
             : this(operation, attribute)
```

Comment:
I took a look at the call point for this and see it's in WebHttpBehavior. I then looked at how Reflector is instantiated and I think we can keep Reflector internal. WebHttpBehavior instantiates Reflector with the parameters `(endpoint.Contract.Namespace, null)`. So instead of instantiating Reflector in WebHttpBinding and saving a reference, store the contract namespace instead. Then modify this constructor to take a string namespace instead of Reflector and instantiate the parent Reflector in the constructor body.

Reply:
This was addressed in a3b94f6c5b7aa036a95ca92f109d95379323a16e.